docs/faq.md

```diff
@@ -0,0 +1,310 @@
+
+
+
+### TROUBLESHOOT
+
+
+
+*How to upgrade?*
+
+Use this command: `pip3 install cryptoadvance.specter --upgrade`
+
+
+*How can I access the web interface if it's hosted on a headless computer?* 
+
+You can either set --host 0.0.0.0 `python -m cryptoadvance.specter server --host 0.0.0.0` or configure nginx to forward connections from specific port to specter.
+ 
+Alternatively, you can also define --port 80 if you want to have it on default http port of the computer.
+
+One drawback though is that with http and **external access** you will not get camera scanning functionality. It is an issue if you are using specter-DIY as it's necessary to scan QR codes with signed transactions. To fix that you will need a self-signed certificate, we have a document on that [here](https://github.com/cryptoadvance/specter-desktop/blob/master/docs/self-signed-certificates.md)
+
+
+*Keep getting: No matching distribution found for cryptoadvance.specter*
+
+*I still get this error when I try to run specter with:*
+`$ python -m cryptoadvance.specter server`
+
+*It states 'no module named cryptoadvance'*
+
+Try `pip3 install cryptoadvance.specter`
+
+Specter only works with python3, so use pip3 to install it
+
+`brew install python3` **(???)**
+
+
+*Even after upgrading to python3 it's still looking at 2.7 version. I uninstalled 2.7, so not sure where to go next?*
+
+Run it with the command `python3 -m cryptoadvance.specter server` - then it will use python3
+
+
+*How to delete a wallet using a remote full node?*
+
+You can't delete the wallet if you are using remote Bitcoin Core node - there is no RPC call to do it remotely. So, deleting wallet works only on the same computer. 
+You can also just delete the wallet manually. It's a folder in `~/.bitcoin` directory and in `~/.specter` as well.
+
+
+
+### DIY TROUBLESHOOT
+
+*Does anyone have any tips on mounting the power bank and QR code scanner to the STM32 board in a somewhat ergonomic manner?*
+
+Use the smallest powerbank possible.
+
+
+### HWW TROUBLESHOOT
+
+
+Got stuck for a second because I wasn't safely removing my SD card reader, so the files were 0 bytes.
+
+
+*With achow's HWI tool, input and output PSBT are the same. And with Electrum 4, I get a rawtransaction, not a base64 PSBT.*
+
+I solved my issue, it turns out my PSBT needed bip32 hints (whatever that means) included. I can now open lightning channels straight from hardware wallet!
+
+
+
+### TECHNICAL QUESTIONS (not dev related)
+
+
+*Does specter-desktop require `txindex=1` to be set in your `bitcoin.conf`?*
+
+No, but you need to enable wallets! `disablewallet=0`
+
+
+
+### FUTURE FEATURES
+
+
+
+*Will specter-desktop ever be a signer?*
+
+Right now it’s only with external wallets, but there is an open issue for hot wallet.
+
```

Comment:
*Will Specter-Desktop ever be a full Hot-Wallet?*

[Comment: Still open for discussion - recently Stepan and me have discussed to keep Specter Desktop as a pure watch-only wallet - and only connect HWWs via USB if necessary. So connect Specter DIY via USB for Coinjoin & Lightning.]

Right now it’s supporting only external wallets, but there is an open issue for hot wallet.
We have plans to add new device type "this computer", see this [issue](https://github.com/cryptoadvance/specter-desktop/issues/58)

At the moment it is more like a coordinator app. For signing we have [specter-diy](https://github.com/cryptoadvance/specter-diy)